IBX-11536: MCP Servers (#3106)
ezsystems/developer-documentation: docs/infrastructure_and_maintenance/security/security_checklist.md

As site administrator, be aware of this when giving editors access to the Page Builder features, and limit that access only to trusted editors.
You can [limit access to specific blocks per content type]([[= user_doc =]]/content_management/configure_ct_field_settings/#default-configuration-of-pages) by defining which page blocks are available to editors.

> ### Activate JWT for MCP, Rest, or GraphQL

Reviewer: ```suggestion
### Activate JWT for MCP, REST, or GraphQL
```

> ### Activate JWT for MCP, Rest, or GraphQL

Reviewer: ```suggestion
### Activate JWT authentication for MCP, Rest, or GraphQL
```

Author: ```suggestion
### Activate JWT authentication for MCP, REST, or GraphQL
```

> If you're using [MCP servers](mcp_guide.md), you must enable JWT for them.

Reviewer: ```suggestion
To use [MCP servers](mcp_guide.md), you must enable JWT authentication  for them.
```

> You can optionally consider enabling JWT for [REST API](rest_api_usage.md) or [GraphQL API](graphql.md) as well.

Reviewer: ```suggestion
You can also consider enabling JWT authentication for [REST API](rest_api_usage.md) or [GraphQL API](graphql.md).
```
"Optionally consider" sounds wrong - either "consider enabling", or "optionally enable"?

> For more information, see how to [enable JWT authentication for specific features](development_security.md#jwt-authentication).

Reviewer 2: ```suggestion
To use [MCP servers](mcp_guide.md), you must enable JWT authentication for them.
You can also consider enabling JWT authentication for [REST](rest_api_usage.md) or [GraphQL](graphql.md) APIs.

For more information, see [Development security](development_security.md#jwt-authentication).
```
